src/Sulu/Component/HttpCache/HandlerInterface.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+namespace Sulu\Component\HttpCache;
+
+use Sulu\Component\Content\StructureInterface;
+use Symfony\Component\HttpFoundation\Response;
+
+interface HandlerInterface
```

Comment:
Does an empty standalone interface really make sense? Shouldn't the other interfaces inherit from it, so that there is at least some structure? Then all the Handlers would not have to additionally implement the HandlerInterface.